src/main/java/com/basho/riak/client/api/commands/kv/MultiCommand.java

```diff
@@ -371,9 +371,9 @@ public void await() throws InterruptedException
         }
 
         @Override
-        public void await(long timeout, TimeUnit unit) throws InterruptedException
+        public boolean await(long timeout, TimeUnit unit) throws InterruptedException
         {
```

Comment:
Nope. If they don't assign/use the return value then the behavior is exactly the same.  If they want to use it though, they can change their code.